client/home/lib/utilities/components/apiaccess/view.coffee

```diff
@@ -42,27 +35,14 @@ module.exports = class View extends React.Component
     <div className='HomeApp-ApiToken--no-token'> No tokens have been created yet. When you create, they will be listed here.</div>
 
 
-  addNewApiToken: ->
-
-    remote.api.JApiToken.create (err, apiToken) ->
-      return showError err  if err
-      TeamFlux.actions.addApiToken apiToken
-
-
-  switchToggle: (state) ->
-
-    TeamFlux.actions.disableApiTokens(state)
-    .catch ({err}) ->
-      showError err
-
   render: ->
 
     toggleState = @props.apiAccessState
 
     <div className='HomeApp-ApiToken'>
 
-      <Toggle checked={toggleState} className='HomeApp-ApiToken--swicth-toggle' callback={@bound 'switchToggle'} />
-      <Header callback={@bound 'addNewApiToken'}/>
+      <Toggle checked={toggleState} className='HomeApp-ApiToken--swicth-toggle' callback={@props.switchToggle} />
```

Comment:
Can you define a `toggleProps` object and use it like `<Toggle {...toggleProps}>` to make it more readable (80 chars per line rule)